### ISSUE TITLE
readthedocs.io url is opencv-python-tutroals not opencv-python-tutorials

### ISSUE THREAD

**Issue by johnrbnsn** [abidrahmank/OpenCV2-Python-Tutorials#213]
readthedocs.io url is opencv-python-tutroals not opencv-python-tutorials. Not sure if this matters or not, found it out when typing in manually. Tutorials is misspelled.

**Comment by madebyollin** [comment]
Yeah... this same issue was raised and closed six years ago (#2) with the response:
>  It is just temporary... anyway the whole site will be taken to docs.opencv.org.

But the RTD site still exists six years later (with the typo) and now consistently outranks https://docs.opencv.org in search results for opencv-related queries:

![image](https://user-images.githubusercontent.com/8733615/76127536-c0b51180-5fb6-11ea-8781-bf27bd196f15.png)

Having a typo in the URL is [common practice for malicious sites trying to impersonate legitimate ones](https://en.wikipedia.org/wiki/Typosquatting), so having the typo in the URL may dissuade cautious users from clicking on what would otherwise be a helpful search result. Please fix!
